packages/anonymous-communication/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@whotracksme/anonymous-communication",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "test": "karma start --single-run --browsers ChromeHeadless karma.conf.cjs"
+  },
+  "author": "Ghostery GmbH",
+  "license": "MPL-2.0",
+  "devDependencies": {
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "chai": "^4.3.6",
+    "fast-check": "^2.23.2",
+    "karma": "^6.3.17",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-rollup-preprocessor": "^7.0.8",
+    "karma-sinon": "^1.0.5",
+    "mocha": "^9.2.2",
+    "rollup": "^2.70.1",
+    "rollup-plugin-polyfill-node": "^0.8.0",
+    "rollup-plugin-sourcemaps": "^0.6.3",
+    "sinon": "^13.0.1"
+  },
+  "dependencies": {
+    "pako": "^2.0.4"
+  },
+  "main": "index.js",
```

Comment:
The preffered way is to set also `exports` field, but as we only distribute ES modules, we can skip the `main` field at all:

Node docs:

> set the main entry point for a package, it is advisable to define both ["exports"](https://nodejs.org/api/packages.html#exports) and ["main"](https://nodejs.org/api/packages.html#main) in the package’s [package.json](https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions) file

More here: https://nodejs.org/api/packages.html#package-entry-points

Reply:
:+1: